src/lists/list-item.jsx

```diff
@@ -198,10 +198,6 @@ const ListItem = React.createClass({
         left: 16,
       },
 
-      primaryText: {
```

Comment:
Well,  as you want. Regarding this component, I thinks that we should make it composable in the futur.